script/install.sh

```diff
@@ -667,67 +708,120 @@ write_env_vars() {
             local dc=gdc
             local dmd=gdmd
             ;;
+        dub*)
+            local libpath=
+            local dc=
+            local dmd=
+            ;;
     esac
 
     logV "Writing environment variables to $ROOT/$1/activate"
     cat > "$ROOT/$1/activate" <<EOF
 deactivate() {
     export PATH="\$_OLD_D_PATH"
+EOF
+    if [ -n "$libpath" ] ; then
```

Comment:
The following code writes thin `activate` scripts that only set the variables that are actually set.
In other words things like `LIBRARY_PATH` aren't written for `dub`